test/string_view-test/string_view-test.cpp

```diff
@@ -0,0 +1,343 @@
+/*
+* string_view-test.cpp
+* Sean Murthy, Ray Chen
```

Comment:
I suggest placing your name first because you did quite a bit of work on this.

```suggestion
* Ray Chen, Sean Murthy 
```